requirements_test_pre_commit.txt

```diff
@@ -1,4 +1,4 @@
-# Automatically generated from .pre-commit-config.yaml by gen_requirements_all.py, do not edit
+# Automatically generated by gen_requirements_all.py, do not edit
```

Comment:
I think the pre-commit file hint here was actually useful. Let's restore that.

Reply:
Done.